validator_test.go

```diff
@@ -11280,3 +11280,67 @@ func TestPostCodeByIso3166Alpha2Field_InvalidKind(t *testing.T) {
 	_ = New().Struct(test{"ABC", 123, false})
 	t.Errorf("Didn't panic as expected")
 }
+
+func TestCustomMessage(t *testing.T) {
+	type test struct {
+		Min      int    `validate:"min=1" message:"test int min"`
```

Comment:
I think I prefer something like this

```
type Test struct {
Min int `validate:"min=1[test int min],max=10[test int max]"`
}
```
If you can implement this, then it will be beautiful.